packages/fluent-theme/src/fluent/styles/Callout.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { ICalloutContentStyleProps } from 'office-ui-fabric-react/lib/Callout';
+import { Depths } from '../FluentDepths';
+import { fluentBorderRadius } from './styleConstants';
+
+export const CalloutContentStyles = (props: ICalloutContentStyleProps) => {
+  return {
+    root: {
+      borderRadius: fluentBorderRadius,
+      borderWidth: 0,
+      boxShadow: Depths.depth16
+    },
+    beakCurtain: {
+      backgroundColor: 'transparent'
```

Comment:
Why do we need this as part of fluent styles? It looks like something that needs to be added to the default styles.
Overwise LGTM. Thanks for fixing the issues.

Reply:
The `beakCurtain` in the default theme has a white background and fills the full width and height of the callout, so it covers up the rounded corners. Rather than making it transparent I've now added the same border radius to it, which gives the same effect and is less likely to cause other issues.